app/graph/types/team_type.rb

```diff
@@ -1,118 +1,121 @@
-  field :pusher_channel, types.String
-  field :search_id, types.String
-  field :search, CheckSearchType
-  field :check_search_trash, CheckSearchType
-  field :trash_size, JsonStringType
+  field :archived, types.Boolean # TODO Rename to 'is_archived'
+  field :private, types.Boolean # TODO Rename to 'is_private'
+  field :avatar, types.String, 'Picture' # TODO Rename to 'picture'
+  field :name, !types.String, 'Team name'
+  field :slug, !types.String, 'Team slug (URL path)'
+  field :description, types.String, 'Team description'
+  field :members_count, types.Int, 'Count of team members'
+  field :projects_count, types.Int, 'Count of team projects (lists)'
+  field :get_slack_notifications_enabled, types.String # TODO Rename to 'slack_notifications_enabled'
+  field :get_slack_webhook, types.String # TODO Rename to 'slack_webhook'
+  field :get_slack_channel, types.String # TODO Rename to 'slack_channel'
+  field :get_embed_whitelist, types.String  # TODO Review and/or rename to 'embed_whitelist'
+  field :get_report_design_image_template, types.String # TODO Merge into 'report_settings'
+  field :search_id, types.String # TODO What's that for?
+  field :search, CheckSearchType, 'Team search'
+  field :check_search_trash, CheckSearchType # TODO Rename to 'search_trash'
+  field :trash_size, JsonStringType # TODO What's that as opposed to `trash_count`?
   field :public_team_id, types.String
   field :permissions_info, JsonStringType
   field :invited_mails, JsonStringType
   field :dynamic_search_fields_json_schema, JsonStringType
-  field :get_rules, JsonStringType
-  field :rules_json_schema, types.String
-  field :rules_search_fields_json_schema, JsonStringType
-  field :medias_count, types.Int
-  field :trash_count, types.Int
-  field :get_languages, types.String
-  field :get_language, types.String
-  field :get_report, JsonStringType
-
-  field :public_team do
-    type PublicTeamType
-
+  field :get_rules, JsonStringType # TODO Rename to 'rules'
+  field :rules_json_schema, types.String # TODO Why not a JsonStringType?
+  field :rules_search_fields_json_schema, JsonStringType # TODO What's this?
+  field :medias_count, types.Int, 'Count of active items'
+  field :trash_count, types.Int, 'Count of trashed items'
+  field :get_languages, types.String # TODO Rename to 'languages'
+  field :get_language, types.String  # TODO Rename to 'language'
+  field :get_report, JsonStringType # TODO Rename to 'report_settings'
+
+  field :public_team, PublicTeamType, 'Public team information' do
     resolve -> (team, _args, _ctx) do
       team
     end
   end
 
-  field :verification_statuses do
-    type JsonStringType
-    argument :items_count, types.Boolean
-    argument :published_reports_count, types.Boolean
+  field :verification_statuses, JsonStringType, 'Verification statuses used in this team' do
+    argument :items_count, types.Boolean # TODO What's this?
+    argument :published_reports_count, types.Boolean # TODO What's this?
```

Comment:
For each status, returns the number of items with that status and the number of published reports with that status, when those arguments are `true`